--- a/board/Marvell/mvebu_armada-8k/mikrotik-rb5009.env
+++ b/board/Marvell/mvebu_armada-8k/mikrotik-rb5009.env
@@ -1,6 +1,6 @@
-openwrt_initramfs=openwrt-mvebu-cortexa72-mikrotik_rb5009-initramfs-uImage.itb
-boot_devices=ubi usb net
-recovery_boot_devices=usb net
+openwrt_initramfs=_INIT_LFS
+boot_devices=ubi net
+recovery_boot_devices=net
 
 button_cmd_0_name=reset
 button_cmd_0=run recovery_bootcmd
@@ -11,9 +11,6 @@
 	led blue:hdr2 on;
 
 	for b in ${recovery_boot_devices}; do
-		if test ${b} = usb; then
-			run usbboot;
-		fi;
 		if test ${b} = net; then
 			run netboot;
 		fi;
@@ -24,9 +21,6 @@
 		if test ${b} = ubi; then
 			run ubiboot;
 		fi;
-		if test ${b} = usb; then
-			run usbboot;
-		fi;
 		if test ${b} = net; then
 			run netboot;
 		fi;
@@ -39,12 +33,6 @@
 	ubi read ${loadaddr} kernel;
 	bootm ${loadaddr};
 
-usbboot=
-	echo Booting from USB Storage;
-	usb start;
-	load usb 0:1 ${loadaddr} ${openwrt_initramfs};
-	bootm ${loadaddr};
-
 netboot=
 	echo Booting from Network;
 	dhcp;
